.github/dependabot.yml

```diff
@@ -4,7 +4,11 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+    labels:
```

Comment:
By setting the `labels` option explicitly, it suppresses the default labels. To preserve those (if that's what you want), the labels `dependencies` and `java` / `github_actions` respectively will need adding here.